go.mod

```diff
@@ -165,18 +166,18 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.16.2
-	k8s.io/api v0.31.6
+	k8s.io/api v0.32.1
 	k8s.io/apiextensions-apiserver v0.31.6
-	k8s.io/apimachinery v0.31.6
+	k8s.io/apimachinery v0.32.1
```

Comment:
NOte we have replace directive above so the version ends up being v0.31.6 as we want.